mantis-runtime/src/main/java/io/mantisrx/runtime/sink/ServerSentEventsSink.java

```diff
@@ -145,7 +146,7 @@ private boolean useSpsc() {
     @Override
     public void call(Context context, PortRequest portRequest, final Observable<T> observable) {
         port = portRequest.getPort();
-        if (runNewSseServerImpl(context.getWorkerInfo().getJobName())) {
+        if (runNewSseServerImpl(context.getWorkerInfo().getJobClusterName())) {
```

Comment:
nit: why did we change this?

Reply:
getJobName() is deprecated and links to use getJobClusterName() , But both getJobName refers the same variable which is jobName string value.
<img width="870" alt="Screenshot 2023-06-29 at 10 36 41 AM" src="https://github.com/Netflix/mantis/assets/16557220/9231dfd8-b015-4275-8377-170e8d5e3496">